README.md

```diff
@@ -191,9 +191,25 @@ Examples:
 setNamespace("MyApplication");
 ```
 
+- **setTimestamp**(Date | number timestamp)
+
+Sets the CloudWatch [timestamp](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#about_timestamp) that extracted metrics are associated with. If not set a default value of `new Date()` will be used.
+
+If set, timestamp will be preserved across calls to flush().
```

Comment:
...for a given `MetricsLogger` instance.

Reply:
Clarified in a subsequent commit.